resources/authgear/templates/en/web/__error.html

```diff
@@ -241,7 +241,12 @@
     {{ else if eq .Error.reason "InvalidWhatsappUser" }}
     <li>{{ template "error-invalid-whatsapp-user" }}</li>
     {{ else }}
-    <li>{{ template "error-unknown" }}</li>
+    <li>
+      {{ template "error-unknown" }}
+      <span id="error-unknown" class="hidden">
+        {{ call $.LogUnknownError .Error }}
+      </span>
```

Comment:
No need to use a span to wrap the template call. `LogUnknownError` always return empty string. So write this instead `{{ template "error-unknown" }}{{ call $.LogUnknownError .Error }}`